lnwallet/channel_test.go

```diff
@@ -827,6 +833,16 @@ func TestForceClose(t *testing.T) {
 			anchorAmt:            anchorSize * 2,
 		})
 	})
+	t.Run("taproot with tapscript root", func(t *testing.T) {
```

Comment:
IIUC these tests only check that the aux tapscript root can co-exist within the channel, but doesn't use the tapscript root in any way

correct?